### PR TITLE
show monocular version in ui

### LINKED ISSUE #316: Show version number in the home page
Now that we have a release process,  it would be useful if we had the version number in the home page.

For example, in the footer we have:

"Monocular is an Open Source project"

We could say something

"This site  is running Monocular v0.3.0
Monocular is an Open Source project"

### PR DESCRIPTION
**Pull request by prydonius**
closes #316

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ before_install:
 install:
   - make -C src/api bootstrap
   - make -C src/ui install
+  - |
+    if [[ -n "$TRAVIS_TAG" ]]; then
```
Comment (prydonius) [inline .travis.yml]: cc @sameersbn this will work as intended, right?

Comment (sameersbn) [inline .travis.yml]: I think you meant to write

```
make -C src/ui VERSION=$TRAVIS_TAG set-version
```

Comment (sameersbn) [inline .travis.yml]: As an aside, if you want to git rid of the inline bash from travis, then you could replace the `VERSION ?= dev` assignment in the Makefile with.

```Makefile
ifeq "$(VERSION)" ""
	override VERSION = dev
endif
```

Comment (prydonius) [inline .travis.yml]: oops good catch, thanks

**Comment by prydonius** [comment]
@sameersbn PTAL!

**Pull request merged**